ladybug.spec.js

```diff
@@ -43,6 +43,26 @@ describe('Ladybug Page tests', function(){
 		teardown();
 	});
 
+	beforeAll(function() {
+		prepare();
+		ladybug.debugTab.click();
+		ladybug.enableReportGenerator();
+		// The test depends on the existence of many Ladybug reports. Create them.
```

Comment:
Geldt dit voor alle tests? Een test moet onafhankelijk kunnen werken. Er mag dus nooit sprake zijn van een volgorde van executie. Mocht je 'dummy' data nodig hebben dan moet dat bij de betreffende test gebeuren, zodat de test slaagt. Nu doe je dit voor het uitvoeren van elke test?

Reply:
De beforeAll() wordt één keer gedraaid. Dit gebeurt voordat de beforeEach() van de eerste test draait.

Ik vind in het algemeen ook dat testen onafhankelijk van elkaar moeten zijn, maar zie ertegenop om hier strict in te zijn met de Protractor test. Ik vind de Protractor test moeilijk om te wijzigen.